bin/setup

```diff
@@ -34,6 +34,8 @@ Dir.chdir APP_ROOT do
   run "cp keys/equifax_rsa.example keys/equifax_rsa"
   run "cp keys/equifax_rsa.pub.example keys/equifax_rsa.pub"
   run "cp certs/saml.crt.example certs/saml.crt"
+  run "gpg --dearmor < keys/equifax_gpg.pub.example > keys/equifax_gpg.pub.bin"
+  system "gpg --batch --import keys/equifax_gpg.example"
```

Comment:
ha! /me looks at old self, *tsk tsk*

I had forgotten that #1269 was never merged and interpreted this as new changes.

I *think* the tests should work w/o needing to import. If we can remove that would be best, IMO.